src/Core/WinSWCore/Extensions/AbstractWinSWExtension.cs

```diff
@@ -16,6 +16,11 @@ public virtual void Configure(IWinSWConfiguration descriptor, XmlNode node)
             // Do nothing
         }
 
+        public virtual void Configure(IWinSWConfiguration descriptor, object yamlObject)
```

Comment:
Would it be possible to pass a real class instead of `object` here? It defeats type checks